src/participant/participant.ts

```diff
@@ -176,20 +177,22 @@ export default class ParticipantController {
       throw new Error('Copilot model not found');
     }
 
-    return await model.sendRequest(messages, {}, token);
+    this._telemetryService.trackCopilotParticipantPrompt(prompt.stats);
```

Comment:
This is going to track for our metadata requests - `namespace` and `intent`. Should we want track those separately, or least add an indicator for them?

Reply:
I've added `internal_purpose` to differentiate namespace and intent requests.